src/utils/Connection.ts

```diff
@@ -36,3 +40,49 @@ export function loadAllConnectionEntities(
     loadConnectionEntities(c, contactEnts, incPeopleEnts)
   );
 }
+
+export function updateConnectionCache(
```

Comment:
this cache maintenance has a very specific effect, yeah?

```suggestion
export function removePendingConnectionRequest
```